test/e2e/suites/unmanaged/unmanaged_functional_test.go

```diff
@@ -1062,6 +1062,21 @@ var _ = ginkgo.Context("[unmanaged] [functional]", func() {
 			})
 			Expect(len(workerMachines)).To(Equal(1))
 			Expect(len(controlPlaneMachines)).To(Equal(1))
+
+			awsCluster, err := GetAWSClusterByName(ctx, namespace.Name, clusterName)
+			Expect(err).To(BeNil())
+
+			// Validate that s3 endpoints were created in the vpc.
```

Comment:
Added a test for deletion, ptal. The unmanaged case, where would you put a check?

Reply:
I don't have a great idea since I'm typically working with unit tests, not E2E. For this networking stuff, there are no unit tests where we could easily mock and test the unmanaged case. We can skip this for now.